src/js/api/documentsApi.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
+
+class DocumenstApi {
+  static getAllDocuments() {
+    return axios
+      .get('http://localhost:3000/documents')
```

Comment:
It's a good practice to keep service host name in some configuration constant so you could easily change it for all application in case you decide to deploy it in another environment.